lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -292,22 +278,26 @@ def add_association(user_attribute, operation_set, object_attribute, policy_mach
     # If no associations are found then the empty array should be returned.
     #
     def associations_with(operation)
-      assocs = operation.policy_element_associations.all(include: [:user_attribute, :operations, :object_attribute])
-      assocs.map { |assoc| [assoc.user_attribute, Set.new(assoc.operations), assoc.object_attribute] }
+
+      assocs = operation.policy_element_associations(true).includes(:user_attribute, :operations, :object_attribute).all
+      assocs.map do |assoc|
+        assoc.clear_association_cache #TODO Either do this better (touch through HABTM on bulk insert?) or dont do this?
```

Comment:
yes and no.  Model cache yes, query cache no.  Maybe thats actually just a yes.  I'll verify.